configure.sh

```diff
@@ -233,11 +230,11 @@ then
 	cd /home/vagrant/torque-sites/competitions/LLIIA2020/etl
 	envsubst < config.py.tmpl > config.py
 	./deploy -g "$DECRYPTION_PASSPHRASE" /home/vagrant/data/decrypted
-	rm -fr /var/www/html/LLIIA2020/extensions/Collection
-	ln -s /home/vagrant/SimpleBook /var/www/html/LLIIA2020/extensions/Collection
+	rm -fr /var/www/html/LLIIA2020/extensions/SimpleBook
+	ln -s /home/vagrant/SimpleBook /var/www/html/LLIIA2020/extensions/SimpleBook
```

Comment:
This seems out of place. I would expect that we would do this for each of the competitions or none of the competitions. Is this somehow left over?